ml_server/main.py

```diff
@@ -0,0 +1,79 @@
+import os
+from pathlib import Path
+from fastapi import FastAPI, HTTPException
+from pydantic import BaseModel
+from typing import List
+from functools import lru_cache
+
+from fastapi import Depends
+from gliner import GLiNER
+
+from tenant_dependency import get_tenant_info
+
+KEYCLOAK_HOST = os.getenv("KEYCLOAK_HOST", "")
+TENANT = get_tenant_info(url=KEYCLOAK_HOST, algorithm="RS256")
+ROOT_PATH = os.environ.get("ROOT_PATH", default="")
+
+
+# Cache will save up to this number of most recently used models. Larger amount will consume more memory,
+# but allows more models to be initialized at the same time.
+NUM_MODELS_TO_CACHE = 3
+
+def get_version() -> str:
+    default = "0.1.0"
+    ver = Path(__file__).parent.parent / "version.txt"
+    if ver.exists() and ver.is_file():
+        with open(ver, "r", encoding="utf-8") as file:
+            line = file.readline().strip()
+            return line or default
```

Comment:
Let's use try except instead of exists ....

Reply:
I copied this function from other services, where it's always the same.